src/deviceDirectory/types.ts

```diff
@@ -22,11 +22,11 @@ export type DeviceStateEnum = "unenrolled" | "cloud_enrolling" | "bootstrapped"
 
 export interface DeviceObject {
     /**
-     * ID of the issuer of the certificate
+     * This is the value of the CN field (in the Subject) of the Issuer of the certificate.
      */
     certificateIssuerId?: string;
     /**
-     * Fingerprint of the device certificate
+     * This is not part of the certificate but rather the SH-256 hash of the certificate.
```

Comment:
This will end up in the SDK docs, does it make sense to a potential user?

Reply:
I think so, we were asked by Joel Goodman how to use these fields and since we found out I thought it was worth updating the docs.

Reply:
OK :)